ci-operator/config/openshift-assisted/assisted-installer-deployment/openshift-assisted-assisted-installer-deployment-master.yaml

```diff
@@ -0,0 +1,40 @@
+base_images:
+  ubi8-python36:
```

Comment:
Why do we need to replace it with a UBI image? this image isn't going to be promoted or used.